quickstart/101-app-service-backup/main.tf

```diff
@@ -0,0 +1,119 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  location = var.resource_group_location
+  name     = random_pet.rg_name.id
+}
+
+resource "random_string" "storage_account_name" {
+  length  = 8
+  lower   = true
+  numeric = false
+  special = false
+  upper   = false
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = random_string.storage_account_name.result
+  resource_group_name      = azurerm_resource_group.rg.name
+  location                 = azurerm_resource_group.rg.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "random_string" "storage_container_name" {
+  length  = 8
+  lower   = true
+  numeric = false
+  special = false
+  upper   = false
+}
+
+resource "azurerm_storage_container" "example" {
+  name                  = random_string.storage_container_name.result
+  storage_account_name  = azurerm_storage_account.example.name
+  container_access_type = "private"
+}
+
+resource "random_string" "service_plan_name" {
+  length  = 8
+  lower   = true
+  numeric = false
+  special = false
+  upper   = false
+}
+
+resource "azurerm_service_plan" "example" {
+  name                = random_string.service_plan_name.result
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  os_type             = "Windows"
+  sku_name            = "S1"
+}
+
+data "azurerm_storage_account_sas" "example" {
+  connection_string = azurerm_storage_account.example.primary_connection_string
+  https_only        = true
+
+  resource_types {
+    service   = false
+    container = false
+    object    = true
+  }
+
+  services {
+    blob  = true
+    queue = false
+    table = false
+    file  = false
+  }
+
+  start  = "2024-01-01"
+  expiry = "2024-12-31"
+
+  permissions {
+    read    = false
+    write   = true
+    delete  = false
+    list    = false
+    add     = false
+    create  = false
+    update  = false
+    process = false
+    tag     = false
+    filter  = false
+  }
+}
+
+resource "random_string" "windows_web_app_name" {
+  length  = 8
+  lower   = true
+  numeric = false
+  special = false
+  upper   = false
+}
+
+resource "azurerm_windows_web_app" "example" {
+  name                = random_string.windows_web_app_name.result
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  service_plan_id     = azurerm_service_plan.example.id
+
+  backup {
+    name                = "Example"
+    storage_account_url = "https://${azurerm_storage_account.example.name}.blob.core.windows.net/${azurerm_storage_container.example.name}${data.azurerm_storage_account_sas.example.sas}&sr=b"
```

Comment:
A question: I saw there's a question mark as delimiter in the url [here](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview#sas-token), do we need a delimiter after `${azurerm_storage_container.example.name}`?

Reply:
@ms-henglu This PR comes from your batch of samples. Could you please reply to Zijie about the delimiter and let me know what I need to change? Thanks.

Reply:
Hi @TomArcherMsft , would you please share me the link of my samples? So I could help investigate. Thanks!

Reply:
@ms-henglu https://github.com/ms-henglu/terraform-provider-azurerm/blob/master/examples/app-service/backup/main.tf

Reply:
Thanks! I just confirmed that in the `data.azurerm_storage_account_sas.example.sas`, it has a prefix "?" . So I think we don't need a delimiter after ${azurerm_storage_container.example.name}

refs: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/vendor/github.com/hashicorp/go-azure-helpers/storage/sas_token.go#L64